tensorly/contrib/decomposition/tests/test_tt_TTOI.py

```diff
@@ -0,0 +1,48 @@
+
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Testing of applying TTOI
+
+@author: Lili Zheng
+"""
+
+import numpy as np
+import math
+import tensorly as tl
+from tensorly import random
+import tensorly.decomposition
+from tensorly.contrib.decomposition.tt_TTOI import TTOI 
+
+def test_TTOI(monkeypatch):
+    """Test for the tt_TTOI function (Tensor train orthogonal iteration)
+    """
+    rng = tl.check_random_state(1234)
+    rank = (1, 1, 1, 1, 1, 1)
+    shape = (20, 20, 20, 20, 20)
+    niter = 2
+    
+    # Generate tensor X with low tensor train rank, and its noisy observation Y
+    X = random.random_tt(shape=shape, rank=rank, random_state=rng, full = True)
```

Comment:
PEP 8 (spaces around =) -- might be worth fixing throughout the code

Reply:
Added spaces around = throughout the code